### ISSUE TITLE
error in missing controller page

### ISSUE THREAD

**Issue by saeideng** [cakephp/cakephp#7986]
hi
with this routes:
```
Router::prefix('manager', function ($routes) {
    $routes->prefix('admin', function ($routes) {

	  // $routes->fallbacks('InflectedRoute');
	   $routes->fallbacks('DashedRoute');

    });
});
```
and with url : projectname/manager/admin/tests
error page display some issues:
![cake-routes-error](https://cloud.githubusercontent.com/assets/8215801/12173935/0456a21a-b56f-11e5-84fd-8564599d09b6.png)

**Comment by jeremyharris** [comment]
Looks like the namespace in the file wrong. You should use IRC or Stack Overflow for questions, this is a big tracker :)

**Comment by saeideng** [comment]
i do not have any question
Missing Controller is wrong

**Comment by jeremyharris** [comment]
I see what you mean now, sorry about that! I mistakenly assumed it matched your class file.

**Comment by markstory** [comment]
Fixed in #7989
